bin/update-docs.rb

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env ruby
+
+require 'yaml'
+require 'active_support/core_ext/hash/deep_merge'
+
+# sed command implementation is different for GNU and macOS
+def sed_i(file)
+  if /darwin/.match?(RUBY_PLATFORM)
+    system "sed -i '' -E 's/[ '$'\t'']+$//' #{file}"
+  else
+    system "sed -i 's/[ \t]*$//' #{file}"
+  end
+end
+
+if ENV['PARALLEL_TESTS_CONCURRENCY']
+  begin
+    system "OPENAPI=1 bundle exec parallel_rspec -n #{ENV['PARALLEL_TESTS_CONCURRENCY']} spec/requests/api/ -o '--seed 1993'"
+    path = './doc/openapi.yaml'
+    file = YAML.safe_load(File.read(path))
+    (2..ENV['PARALLEL_TESTS_CONCURRENCY'].to_i).each do |number|
+      file.deep_merge!(YAML.safe_load(File.read("./doc/openapi#{number}.yaml")))
+    end
+    File.write(path, YAML.dump(file))
```

Comment:
this is minor, but `file` is a `Hash` so maybe we can sort the examples that should be somewhere inside it, so docs get somewhat a bit better grouped together?
![image](https://github.com/rootstrap/rails_api_base/assets/6373536/cb186f87-98fe-415a-84b1-4bfdb5d10430)

Reply:
nice!